lib/components/atoms/ListItem/ListItem.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+/**
+ *
+ * ListItem
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+
+import styles from './ListItem.style';
+import type { Props } from './types';
+
+const ListItem = ({ className, children }: Props): Node => {
```

Comment:
Why is this component not a child of list or carousel. We can't use it without List